omnibus/files/private-chef-cookbooks/private-chef/recipes/opscode-erchef.rb

```diff
@@ -55,7 +57,9 @@
                                                                  :actions_exchange => actions_exchange,
                                                                  :ldap_encryption_type => ldap_encryption_type,
                                                                  :solr_elasticsearch_major_version => elastic_search_version,
-                                                                 :helper => OmnibusHelper.new(node)))
+                                                                 :helper => OmnibusHelper.new(node))
+     }
+   }
```

Comment:
If the thing on line 51 is a hash, I think the curly brackets on lines 50 and 61 are unnecessary.